File: tests/openmp_examples/devices/device.4.c
#pragma omp begin declare variant match(device={kind(host)})
#pragma omp end declare variant
#pragma omp begin declare variant match(device={kind(nohost)})
#pragma omp end declare variant
#pragma omp target is_device_ptr(mem_dev_cpy) device(t)
#pragma omp teams distribute parallel for

File: tests/openmp_examples/devices/device.4.f90
!$omp   target is_device_ptr(cp) device(targ_dev)
!$omp   end target
!$omp    target device(targ_dev) is_device_ptr(cp_dst)
!$omp    end target
!$omp    target device(targ_dev) is_device_ptr(cp_dst)
!$omp    end target
!$omp    target device(targ_dev) map(h_fp)
!$omp    end target
